docs/guide/plugins.md

```diff
@@ -0,0 +1,202 @@
+---
+sidebar: auto
+---
+
+# Plugins
+
+## Creating plugin
+
+Plugin development can be easily made locally. First, we need to choose where would we want to plug into. To see all available plugin slots we may want to select `yes` during `shopware-pwa init` command when it asks to allow dev mode.
+
+![cli plugin dev mode](./../assets/cli-plugin-dev-mode.png)
+
+It shows then a button in the footer, which allows us to turn on Dev mode, and see all plugin slots in theme.
+
+![turn-on-dev-mode](./../assets/turn-on-dev-mode.png)
+
+it will show us available slots on the theme, and you can click on any of them to see its name in the console
+
+![theme-slots](./../assets/theme-slots.png)
+
+### Local plugin
```

Comment:
Maybe we can also build the bridge to the Shopware instance plugins here, saying that `shopware-pwa init` will take the resources from each plugins `Resources/app/pwa` directory and place them within sw-plugins, so people will understand why they end up in this directory. Obviously in your example, they are writing the plugins directly into `sw-plugins` but it should be clear, that this is also the connection point to SW-Plugins.

Reply:
https://github.com/DivanteLtd/shopware-pwa/pull/695/commits/bf2f4561abada74ea64364c83ecc4ec8028de7d8

Reply:
is that okay?

Reply:
Yup, should do it!